homework-submission/week4/src/TodoList.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+
+class TodoList extends Component {
+  render() {
+    const doneStyle = { textDecoration: "line-through" };
+    const unDoneStyle = {};
+    const { deadline, description, done, id } = this.props.todoItem;
+
+    return (
+      <div>
+        <input
+          id={id}
+          type="checkbox"
+          checked={done}
+          onChange={this.props.onChange}
+        />
+        <span style={done ? doneStyle : unDoneStyle}>
```

Comment:
Using `style` is a nice idea But it would be More profession look if you used Css Classes, you can follow [__This__](https://github.com/HackYourFuture/React/pull/161#discussion-diff-199063438R22)